config/initializers/permissions.rb

```diff
@@ -127,7 +127,7 @@
                    require: :loggedin
 
     wpt.permission :edit_work_packages,
-                   { issues: [:edit, :update, :update_form],
+                   { issues: [:edit, :update],
                      :'work_packages/bulk' => [:edit, :update],
                      work_packages: [:edit, :update, :new_type,
                                      :preview, :quoted],
```

Comment:
What about those here:
`:new_type, :preview, :quoted`
They are not used in the API v2 so they should be removable.